health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/UpdateChecklist.js

```
@@ -515,16 +560,28 @@
+                                    type={"button"}
+                                    size={"large"}
+                                    variation={"primary"}
+                                    label={t("CREATE_CHECKLIST")}
+                                    onClick={() => {
+                                        const processed = organizeQuestions(tempFormData);
+                                        const { local: generatedLocal } = generateCodes(processed);
+                                        // setLocalisationData(generatedLocal);
+                                        setShowLocalisationPopup(true);
+                                        setShowPopUp(false);
+                                    }}
+                                />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)